example_annotate.py

```diff
@@ -49,7 +51,12 @@
         dataset = PMIDReader(args.pmids).read()
     elif os.path.exists(args.dir_or_file):
         print(warning)
-        dataset = TextFilesReader(args.dir_or_file).read()
+        #dataset = TextFilesReader(args.dir_or_file).read()
+        dataset = HTMLReader(args.dir_or_file, whole_basename_as_docid=True, hdfs_url=None, hdfs_user=None).read()
+        file_html=args.dir_or_file
+        file_json=file_html.replace('.html','.json')
+        AnnJsonAnnotationReader(file_json, read_only_class_id=None, whole_basename_as_docid=True, hdfs_url=None, hdfs_user=None).annotate(dataset)
+       
```

Comment:
Don't we need to call the function `validate_entity_offsets` somewhere?

Reply:
the AnnJsonAnnotationReader calls the validate_entity_offsets itself.